website/src/pages/guides/getting-started.mdx

```diff
@@ -104,14 +104,15 @@ To make Biome the default formatter open a supported file (JavaScript or TypeScr
 
 ### Other Editors
 
-We would love to support more editors, but we don't have the capacity to implement and maintain multiple editor integrations at the moment. You can help us prioritize by [voting](https://github.com/biomejs/biome/discussions/3544) for your favourite editor. If you're interested in building an integration for Biome, please [reach out](https://github.com/biomejs/biome/issues/2390), and we would be more than happy to support you.
-
-If you are looking for editor support in a JetBrains IDE like WebStorm, then visit the relevant [issue](https://youtrack.jetbrains.com/issue/WEB-46895/Support-for-Romejs) to upvote the ticket.
+We would love to support more editors, but we don't have the capacity to implement and maintain multiple editor integrations at the moment.
+Because Biome uses the language server protocol (LSP), it should be relatively easy to get Biome working with any editor that has an LSP client.
+If you're interested in building an integration for Biome, please, reach out, and we would be more than happy to support you.
```

Comment:
```suggestion
We would love to support more editors, but we don't have the capacity to implement and maintain multiple editor integrations at the moment.
Because Biome uses the language server protocol (LSP), it should be relatively easy to get Biome working with any editor with an LSP client.
If you're interested in building an integration for Biome, please, reach out, and we would be more than happy to support you.
```